packages/core-connector-interface/src/index.d.ts

```diff
@@ -101,6 +101,8 @@ declare interface HoprCoreConnector {
    */
   withdraw(currency: Currencies, recipient: string, amount: string): Promise<string>
 
+  hexAccountAddress(): Promise<string>
```

Comment:
What's the purpose of this method?

Reply:
in a lot of places we look up the account address as a hex string. This provides an easy way for core.